examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala

```diff
@@ -17,11 +17,6 @@
 
 package org.apache.spark.examples.graphx
 
-import org.apache.spark.SparkContext._
```

Comment:
Yes, but, why remove the description, which already existed in the help message in this file?